package.json

```diff
@@ -45,7 +45,8 @@
     "android": "npm run android:emulator & npm run android:build",
     "spellcheck": "mdspell -r --en-gb --ignore-acronyms --ignore-numbers --no-suggestions '*.md' '**/*.md' '!**/node_modules/**/*.md' '!ios/Pods/**/*.md'",
     "spellcheck:interactive": "mdspell --en-gb --ignore-acronyms --ignore-numbers '*.md' '**/*.md' '!**/node_modules/**/*.md' '!ios/Pods/**/*.md'",
-    "start": "react-native start"
+    "start": "react-native start",
+    "fix:package-lock": "git status -s | grep package-lock.json | awk '{print $2}' | xargs sed -i 's/http:\\/\\//https:\\/\\//g'"
```

Comment:
The `-i` expects input

```suggestion
    "fix:package-lock": "git status -s | grep package-lock.json | awk '{print $2}' | xargs sed -i '' 's/http:\\/\\//https:\\/\\//g'"
```

Reply:
At the moment it is interpretting the regex as the input for `-i` and doing this

![Screenshot 2019-03-15 at 15 45 34](https://user-images.githubusercontent.com/30267516/54443837-641f6d80-4739-11e9-8ace-50fa7a676f28.png)

Reply:
Are you sure? `-i` as far as I know is `in-place`, meaning to replace the file

Reply:
Ah, nvm this is that problem with mac os using an old version of sed

Reply:
```
    -i extension
             Edit files in-place, saving backups with the specified extension.  If a zero-length extension is given, no backup will be saved.  It is not recommended to give a zero-length extension when in-place editing files, as you risk corruption or partial content in situations where disk space is exhausted, etc.
```

Reply:
Yeah, but that is not the behaviour with GNU seed (which I have locally and is by default on linux) and they are not compatible

Reply:
So I've replaced this with `replace` (no pun intended) and it works now. I had to remove from post install though as it is not working. I suspect the package lock is only saved on the disk after the post install script runs